### DISCUSSION TITLE
Theme is broken

### DISCUSSION THREAD

**Discussion by AltriaChan** [JanDeDobbeleer/oh-my-posh#4298] [Troubleshoot] [answered]
![스크린샷 2023-09-29 065406](https://github.com/JanDeDobbeleer/oh-my-posh/assets/59331917/b8894650-1b8f-4829-81ea-59d4ab31bf69)
How to remove these dark triangles?
Originally, it was only visible when using complex themes in vscode, but since resetting Windows, it is always visible even in the Windows terminal.

**Comment by JanDeDobbeleer** [comment]
@tryfresh you need to change the rendering engine in Windows Terminal.

**Reply by AltriaChan** [reply]
![스크린샷 2023-09-29 170842](https://github.com/JanDeDobbeleer/oh-my-posh/assets/59331917/187bcef7-6ea7-4033-8407-0cb1e172bb85)

Hmm... I've tried all eight possible cases with those three buttons, and it's still the same.

![스크린샷 2023-09-29 170852](https://github.com/JanDeDobbeleer/oh-my-posh/assets/59331917/d89422d1-fe0b-4798-aa29-538897bf00ce)

**Reply by JanDeDobbeleer** [reply, marked as answer]
It's definitely a terminal issue as we use an ANSI `hack` that allows transparent foreground. Windows Terminal didn't support that (but I thought they did implement it).

**Comment by AltriaChan** [comment]
There was an issue because it was set to Always. Fixed.
![스크린샷 2023-09-30 050013](https://github.com/JanDeDobbeleer/oh-my-posh/assets/59331917/1fad6365-0d23-4f5a-b9d4-ec141beedc0e)
![스크린샷 2023-09-30 050352](https://github.com/JanDeDobbeleer/oh-my-posh/assets/59331917/55cbe8d0-5c0d-482e-bc1d-bc5fa07201af)
